templates/default/common_scripts.html.twig

```diff
@@ -75,13 +75,16 @@
                     latitude: _latlng.lat,
                     longitude: _latlng.lng
                 },
-                {% include "elements/js/loader.js.twig" %},
+                {# FIXME : Is a loader really necessary here ?
```

Comment:
Since data are loaded form an ajax call, yes it is. Most of the time, we're using a local server to develop, and load are instant. But on some servers/configurations, this may take much more time, therefore we have to inform the user something is happening.

Reply:
OK. With the current style, it creates a blink effect between modals.
This issue will have to be handled in the theme or maybe using [multiple modals](https://fomantic-ui.com/modules/modal.html#multiple-modals).

Reply:
Hum... right now, we first made the ajax call (loader required) and then display full modal.
Maybe a better solution would be to create the modal with a SUI loader; then make ajax call?

Reply:
I'm wondering if the "blink" effect is due to the instant load of the ajax call. In that case, a small timer could also to the trick.

Reply:
"Visually", it is due to the transition applied on the [dimmers](https://fomantic-ui.com/modules/dimmer.html).
The loader and each modal use one.
The most direct solution would be to "play" with the dimmers.
But I presume the cleanest solution would be to use multiple modals the SUI way 😏

Reply:
Ok, but anyways, I guess this is the same problem for all core modals aswell... This should be done in a separate PR